mmv1/products/securitycentermanagement/product.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: SecurityCenterManagement
+display_name: Security Command Center Management (SCC)
+legacy_name: scc_management
```

Comment:
To add to my previous comment, the resource name is also impacted by this value

Reply:
I have changed the resource name in the config, test case and example files. So I think the `legacy name` shouldn't affect  the resource now, is that right?

Reply:
I believe legacy name controls this section of the resource type's name: google_**scc_management**\_folder_security_health_analytics_custom_module.

Everything's ok with the legacy name currently and you don't need to change it; I highlighted legacy_name in a comment in case you wanted the resource type name to start with google_**scc_management_organization**\_ , because then we'd need to decide whether to achieve that by updating `legacy_name` in product.yaml or `name` in FolderSecurityHealthAnalyticsCustomModule.yaml.

Given the change in https://github.com/GoogleCloudPlatform/magic-modules/pull/10839/commits/d0b8ac7a1bf685d8292378759202743caa8383c3#diff-908bcb8f9291d65c4bb97b2f734cea4d65ef3b6310e12a580d5f3196cb696e9a this isn't necessary

Reply:
Cool, thanks.